ait/core/table.py

```diff
@@ -456,11 +456,57 @@ def __init__(self, filename=None):
         self.fswtabdict = None
 
     def dirty(self):
-        return not os.path.exists(self.pcklname) or os.path.getmtime(
-            self.filename
-        ) > os.path.getmtime(self.pcklname)
+        """True if the pickle cache needs to be updated, False to use pickle binary"""
+        return self.check_yaml_timestamps(self.filename, self.cachename)
+
+    def check_yaml_timestamps(self, yaml_file_name, cache_name):
```

Comment:
Does this function already exist under util.py?  Can't table.dirty just call that?